### PR TITLE
fix: suggested changes

### PR DESCRIPTION
**Pull request by amarphule**
Made changes according to suggestions
1. Improved font size and spacing according to mobile view.
2. Added success validation style for input.
3. Added alt text in images.
4. Added favicon to website.
5. Just added a dark theme toggle ( the theme is not changing color)
6. Improved responsiveness.
7. Added scroll to top button.
8. Added Modal dismiss functionality.
9. For the heading I don't want to change default tags, that's why made it with a paragraph tag.

### CODE REVIEW

**Review by SanjayJatti — APPROVED** [review]

**Comment by SanjayJatti** [comment]
code is clean

File: components/input.css [hunk 1]
```diff
@@ -3,25 +3,30 @@
...
-  justify-content: space-between;
+  justify-content: space-around;
   border: 1px solid var(--secondary-color);
 }
-.label {
-  font-size: 1.4rem;
-}
+/* .label {
+  font-size: 1rem;
+} */
```
Comment (abhijeetekad) [inline components/input.css:11]: Remove commented code.

File: js/modal.js [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
-
+let action = document.querySelector(".action");
+let modal = document.querySelector(".modal");
+let close = document.querySelector(".close");
+let closebtn = document.querySelector("#close");
 
+// closebtn.addEventListener("click", () => {
+//   //   modal.style.display = "none";
+// });
+action.addEventListener("click", () => {
```
Comment (abhijeetekad) [inline js/modal.js:9]: +1

File: style.css [hunk 3]
```diff
@@ -25,19 +25,87 @@
...
+}
+
+/* Scroll to top */
+
+/* Start Theme mode */
+/* body.dark {
+  background-color: var(--primary-color);
+  color: white;
+} */
+
```
Comment (abhijeetekad) [inline style.css:61]: +1

**Review by abhijeetekad — APPROVED** [review]

**Pull request closed without merging**